File: 3223-minimum-length-of-string-after-operations/3223-minimum-length-of-string-after-operations.cpp
class Solution {
public:
    int minimumLength(string s) {
        int ans=0;
        unordered_map<char,int>m;
        for(char x:s){
            m[x]++;
        }
        for(auto const &p:m){
            if(p.second%2==0){
                ans+=2;
            }
            else ans+=1;
        }
        return ans;
    }
};

File: 3223-minimum-length-of-string-after-operations/3223-minimum-length-of-string-after-operations.py
class Solution(object):
    def minimumLength(self, s):
        """
        :type s: str
        :rtype: int
        """
        ans = 0
        m = {}
        for x in s:
            m[x] = m.get(x, 0)+1 

        for x in m.values():
            if x % 2 == 0:
                ans += 2
            else:
                ans += 1

        return ans
